package.json

```diff
@@ -339,6 +339,7 @@
     "sinon": "7.5.0",
     "style-loader": "0.23.1",
     "styled-components": "5.3.9",
+    "teamcity-service-messages": "0.1.14",
```

Comment:
The only thing we're using it for is [blocks of messages](https://www.jetbrains.com/help/teamcity/service-messages.html#Blocks+of+Service+Messages) so we could probably implement this ourselves if we wanted to drop this dependency. 

Given this [depends](https://github.com/pifantastic/teamcity-service-messages/blob/2afe888d589adead5df0611a1420d711470c71eb/package.json#L16) on an [unmaintained dependency](https://github.com/caolan/nodeunit#deprecated-project) would it be possible to remove this and just implement the blocks of messages ourselves in a separate function?

Reply:
@kdenhartog why do we care about dev dependencies in external libs? from my understanding they should not be installed in our environment.

the `teamcity-service-messages` library doesn't have any "production" dependencies, so I decided to use it as is and tbh I'd like to just keep it as is.

Reply:
The main reason I was thinking about was just to keep the number of dependencies in use minimal because this limits the supply chain concerns of dev devices. It's not a massive deal and certainly not a critical concern and when I was weighing it against the usage I figured it was worth at least bringing up the risk. This certainly isn't critical so I'm good with just approving as is as well. Sorry for not communicating that more clearly in the previous message.